_providers/manitu.de.md

```diff
@@ -0,0 +1,7 @@
+---
+domains: manitu.de
+last_checked: 2019-04
+name: Manitu (Custom Domain)
```

Comment:
Is this for custom domains or for manitu.de?

Reply:
There are of course some manitu.de addresses, e.g. support@manitu.de, but they don't really seem to be a mail provider. I took them out.